### ISSUE TITLE
Loading logged in instance of develop.element.io takes significantly longer than usual due to broken matrix.org

### ISSUE THREAD

**Issue by MTRNord** [element-hq/element-web#27374]
### Steps to reproduce

1. Open develop.element.io with a HS that doesnt properly respond on /versions

#### What did you expect?

Since I am logged in it should just fetch my own HS and not the version or auth_issuer endpoints of matrix.org which are given in the config.json

#### What happened instead?

It blocked on requesting the version and auth_issuer endpoints on matrix.org ending in `Error processing homeserver config: Object { state: "FAIL_ERROR", error: "Homeserver URL does not appear to be a valid Matrix homeserver", base_url: "https://matrix-client.matrix.org/" }`

### Operating system

Fedora Silverblue 40

### Browser information

Firefox 124.0.1 (64-Bit)

### URL for webapp

develop.element.io

### Application version

Version von Element: 3cb5b840ce84-react-6dd6a7697c2d-js-6fedda91f9d2 Krypto-Version: Rust SDK 0.7.0 (61b175b), Vodozemac 0.5.1

### Homeserver

matrix.midnightthoughts.space

### Will you send logs?

Yes

**Comment by t3chguy** [comment]
Duplicate of https://github.com/element-hq/element-web/issues/11655
